ancpbids/plugins/plugin_dsvalidator.py

```diff
@@ -17,9 +18,12 @@ def execute(self, dataset, report: ValidationPlugin.ValidationReport):
                 val = getattr(obj, name)
                 use = member['use']
                 if (lb > 0 or use == 'required') and not val:
-                    report.error(f"Missing required field {name} at {top_path}.")
+                    # TODO: Identify more precisely where these errors emerge from
+                    report.error(f"Missing required field {name} at {top_path}.",
+                                 f)
```

Comment:
good idea to also add a reference to the offender, which in this case should be obj